server/tests/controllers/comments.js

```diff
@@ -118,6 +148,82 @@ describe('Comment Controller', () => {
           done();
         });
     });
+
+    // for highlighted comment
+    it('should return error if articleBody is missing', (done) => {
+      request(app)
+        .post('/api/articles/test-article-slug12345/comments')
+        .set('Authorization', token)
+        .send(highlightedComment.noArticleBody)
+        .end((err, res) => {
+          expect(err).to.not.exist;
+          expect(res.type).to.equal('application/json');
+          expect(res.status).to.equal(400);
+          expect(res.body.status).to.equal('error');
+          expect(res.body.error.articleBody).to.equal('injected article body missing');
+          done();
+        });
+    });
+
+    it('should return error if highlighted text is missing', (done) => {
+      request(app)
+        .post('/api/articles/test-article-slug12345/comments')
+        .set('Authorization', token)
+        .send(highlightedComment.noHighlightedText)
+        .end((err, res) => {
+          expect(err).to.not.exist;
+          expect(res.type).to.equal('application/json');
+          expect(res.status).to.equal(400);
+          expect(res.body.status).to.equal('error');
+          expect(res.body.error.highlighted).to.equal('highlighted text missing');
+          done();
+        });
+    });
+
+    it('should return error if cssId is missing', (done) => {
+      request(app)
+        .post('/api/articles/test-article-slug12345/comments')
+        .set('Authorization', token)
+        .send(highlightedComment.noCssId)
+        .end((err, res) => {
```

Comment:
Object destructuring makes your code more concise. You can apply this on lines `189-193`
const { type, status, body } = res;
You can even go a step further and do everything in one destructuring pattern
`const { type, status, body: {status}, body: { error: { cssId } } } = res;` 
But if it becomes less readable and hairy, simple destructuring would do.

Reply:
Thank you Emmanuel. We used the dot accessor `res.type` and `res.body.status` for consistency within the comments tests file. However, I do think object destructuring is more cryptic in this case. Do you think we should proceed with this change?